src/test/scala/backgammon/BackgammonTest.scala

```diff
@@ -13,9 +14,13 @@ class BackgammonTest extends Specification with ValidatedMatchers {
       vg.flatMap { g => g.apply(action).map(_._1) }
     }
 
-  def playActionStrs(actionStrs: List[String], game: Option[Game] = None): Validated[String, Game] =
+  def playActionStrs(
+      actionStrs: List[String],
+      game: Option[Game] = None,
+      variant: Option[Variant] = None
+  ): Validated[String, Game] =
     playUciList(
-      game.getOrElse(Game.apply(variant.Backgammon)),
+      game.getOrElse(Game.apply(variant.getOrElse(Variant.default))),
```

Comment:
Is there a difference here between writing Game.apply(<some_param>) and Game(<some_param>) ?
If not, do you prefer I write the apply in the tests I am currently writing for Abalone ?

Reply:
Both `Game.apply(variant)` and `Game(variant)` work here. Personally I prefer `Game()` to be used when you are instantiating an instance of `Game` with its paramter-set and call `Game.apply` when you are explicitly instantiating a `Game` through an `apply` method (in this case https://github.com/Mind-Sports-Games/strategygames/blob/23c42bbbe09a53d0376a0ef4117d5223fc8c682d/src/main/scala/togyzkumalak/Game.scala#L75).
Despite that being my personal preference, I don't think this standard is followed throughout the codebase and I have no problem with you doing either approach